projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -28,6 +28,12 @@
   void setCompletion(RoleCompletion<? super ContextT, SourceT> completion);
   void setDeleteHandler(DeleteHandler handler);
   void setClipboardParameters(ContentKind<SourceT> kind, Function<SourceT, SourceT> cloner);
+  <ContentT> void supportContentKind(ContentKind<ContentT> kind,
+      Function<ContentT, SourceT> fromKind,
+      Function<SourceT, ContentT> toKind);
+  <ContentT> void supportContentKindCompatibleToItemsList(ContentKind<ContentT> kind,
```

Comment:
Please see https://github.com/JetBrains/jetpad-projectional/pull/185#discussion_r57545199. That's not a List content kind, this is something can be mapped to List of items.